server/api/users.js

```diff
@@ -1,5 +1,5 @@
 const router = require('express').Router();
-const { User, Order, OrderItem, Product } = require('../db');
+const { User, Order, OrderItem, Product, ProductVariant } = require('../db');
```

Comment:
`Product` was there before, but is not being used
```suggestion
const { User, Order, OrderItem, ProductVariant } = require('../db');
```